opaque_keys/__init__.py

```diff
@@ -357,6 +402,8 @@ def __setstate__(self, state_dict):
             if key in self.KEY_FIELDS:  # pylint: disable=no-member
                 setattr(self, key, state_dict[key])
         self.deprecated = state_dict['deprecated']  # pylint: disable=assigning-non-slot
+        self._unicode = None  # pylint: disable=assigning-non-slot
```

Comment:
I honestly don't know. I thought this might fall in the category of "pylint gets a little wonky when you start doing evil things". I'll check again, but I'm pretty sure I added this in direct response to a pylint error message.